src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -44,8 +47,45 @@ public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term,
             this.term = term;
             this.qualifier = qualifier;
             this.value = value;
+            this.usesDefault = false;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EdmVocabularyAnnotation class
+        /// to the default value.
+        /// </summary>
+        /// <param name=“target”>Element the annotation applies to.</param>
+        /// <param name=“term”>Term bound by the annotation</param>
+        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term)
+        {
+            // Check arguments
+            EdmUtil.CheckArgumentNull(target, "target");
+            EdmUtil.CheckArgumentNull(term, "term");
+            if (term.DefaultValue == null)
+            {
+                throw new InvalidOperationException(Strings.EdmVocabularyAnnotations_DidNotFindDefaultValue(term.Name));
+            }
+
+            // Initialize
+            this.target = target;
+            this.term = term;
+            this.qualifier = null;
+            this.value = EdmTermExtensions.BuildDefaultEdmExpression(term.Type, term.DefaultValue);
+            this.usesDefault = true;
         }
 
+        /* Note: This constructor is ambiguous with the other three-param constructor.
+        /// <summary>
+        /// NEW: Initializes a new instance of the EdmVocabularyAnnotation class 
+        /// to the default value.
+        /// </summary>
+        /// <param name=“target”>Element the annotation applies to.</param>
+        /// <param name=“term”>Term bound by the annotation</param>
+        /// <param name=“qualifier”>Qualifier used to distinguish bindings.</param>
+        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term,
+        string qualifier)
+        { }*/

```

Comment:
so what is the solution if someone wants to build an EdmVocabularyAnnotation with a qualifier that uses a default value?

Reply:
Ah yeah, I must have missed this from my draft -- I had been wondering if someone had any advice there, but I have an idea I will submit.